storage/filter_helper_test.go

```diff
@@ -1,7 +1,7 @@
 package storage
```

Comment:
Revert the changes in this file :)

Reply:
Apart from reverting this, I have also changed the expected nil values to match the empty policies/roles that is returned now.